app/src/main/java/com/github/siela1915/bootcamp/MainHomeActivity.java

```diff
@@ -163,10 +165,16 @@ protected void onCreate(Bundle savedInstanceState) {
                     setContainerContent(R.id.fragContainer,AboutPageFragment.class,false);
                     break;
                 case R.id.menuItem_login:
-                    setContainerContent(R.id.fragContainer,ProfileFragment.class,false);
+                    intent.putExtra("com.github.siela1915.bootcamp.navToProfile", true);
```

Comment:
It can be more maintainable if this string is stored in a final var or somewhere more central like a property file. And as I see that `navToProfile` is also used elsewhere in this file, so you may consider breaking it into two string and storing them separately. :)